benchmarks/ekomark/benchmark/external/LHA_utils.py

```diff
@@ -19,10 +19,16 @@
 toy_xgrid = np.array([1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 0.1, 0.3, 0.5, 0.7, 0.9])
 
 # list
-raw_label_list = ["u_v", "d_v", "L_m", "L_p", "s_p", "c_p", "b_p", "g"]
+raw_label_list = {
+    "default": ["u_v", "d_v", "L_m", "L_p", "s_p", "c_p", "b_p", "g"],
+    "NNLO_FFNS": ["u_v", "d_v", "L_m", "L_p", "s_v", "s_p", "c_p", "g"],
```

Comment:
- I don't like the repetition in the code ... 
- I'd rather either repeat some zeros in the data file (as I've been already doing with c_+ and b_+)
- or adjust the data in `compute_LHA_data` where all informations are present

Reply:
Good you addressed this in e764625  but can we not do this more consistently? i.e. also remove this https://github.com/N3PDF/eko/pull/66/commits/e764625564fc5e3c1ef5e1ca024f122c524510c5#diff-552b6e67715034c2e3566ba07603a7f1e15d554ab40879d61393b7b41eb9ab75R45 and this https://github.com/N3PDF/eko/pull/66/commits/e764625564fc5e3c1ef5e1ca024f122c524510c5#diff-552b6e67715034c2e3566ba07603a7f1e15d554ab40879d61393b7b41eb9ab75R80 (as Alessandro already said already, I think ...) - maybe even remove `LHA_rotate_to_evolution` as a whole?

Reply:
To remove this function we will need to add the LHA basis to ``eko.basis_rotation`` , would this way be better ?

Reply:
No, not remove `rotate_data` - just the double `LHA_to_flavor` and `LHA_to_evol' since one of them, say `_to_flavor`, should be enough

Reply:
And remove the special treatment of FFNS NNLO if possible ...